Lib/unittest/async_case.py

```diff
@@ -90,9 +90,13 @@ def _callSetUp(self):
         self._callAsync(self.asyncSetUp)
 
     def _callTestMethod(self, method):
-        if self._callMaybeAsync(method) is not None:
-            warnings.warn(f'It is deprecated to return a value that is not None from a '
-                          f'test case ({method})', DeprecationWarning, stacklevel=4)
+        result = self._callMaybeAsync(method)
+        if result is not None:
+            msg = (
+                'It is deprecated to return a value that is not None '
```

Comment:
Nitpick:
```suggestion
                f'It is deprecated to return a value that is not None '
```
I looks better if every line has the same prefix.